dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -74,7 +74,7 @@ protected String parseCommandText(String commandText) {
                     paramName = "";
                 String dataType = rs.getString(2);
                 DbParameterAccessor dbp = createDbParameterAccessor(
-                        paramName,
+                		'"'+paramName+'"',
```

Comment:
Is it necessary to enclose that in `"` here? So far the behavior has been to store the original name in the parameter accessors.